addons/gr/mydata/extensions.go

```diff
@@ -13,8 +13,7 @@ const (
 	ExtKeyIncomeCat    = "gr-mydata-income-cat"
 	ExtKeyIncomeType   = "gr-mydata-income-type"
 	ExtKeyPaymentMeans = "gr-mydata-payment-means"
-
-	InvoiceTypeRetailPrefix = "11."
+	ExtKeyOtherTax     = "gr-mydata-other-tax"
```

Comment:
"Other tax" sounds really confusing... I'm wondering if "miscellaneous" would be a better translation: `gr-mydata-misc-tax`?

The title in English could be "Miscellaneous or other taxes category".